doc/iam.md

```diff
@@ -0,0 +1,6 @@
+# Ignition and IAM roles
```

Comment:
Please move this into another document. Having a ton of small files is very difficult to traverse.